src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/documents/recallnotification/LetterToProbationGenerator.kt

```diff
@@ -20,16 +22,25 @@ class LetterToProbationGenerator(
       setVariable("teamName", RECALL_TEAM_NAME)
       setVariable("teamContactNumber", RECALL_TEAM_CONTACT_NUMBER)
       setVariable("licenceRevocationDate", context.licenceRevocationDate.asStandardDateFormat())
-      setVariable("recallLengthDescription", context.recallLengthDescription.asFixedTermLengthDescription())
+      if (context.recallType == FIXED) {
```

Comment:
Fix, not specific to ticket?

Reply:
Well, it was an optional value, that I wanted to make non-optional so it blows up rather than just inserts empty data into the thymeleaf doc.